yarn.lock

```diff
@@ -0,0 +1,5493 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file is obviously quite large. Scanning through it, I see a bunch of dependencies that I can't imagine we are actually using, e.g. AWS libraries. Is this file going to get trimmed down in a later commit I haven't reached yet?

Reply:
Now, several comments in, I am understanding that I did not have the right expectations about how big the dependency graph truly is for a modern JS app. Feel free to ignore this comment unless there's something in here you'd like to respond to.

Reply:
Yeaaaaaaaah. This is normal. The size of the lock file isn't really much of a concern, that's just all of the libraries that get installed locally; the majority are only used in development. However, we do want to be cognizant of our production bundle size as that has very real ramifications for performance. This is why I configured the bundle analysis plugin for webpack, so that we can keep an eye on this over time. We could even hook this into CI and generate alerts (or block builds) if bundle size increases over some threshold.

One way we can help manage bundle size is to use webpack's tree-shaking feature in the future. This is one of a handful of optimizations I'm eyeing to file as a 'Production webpack build optimization' issue. My general sense is that bundle size may never be an issue for `recdiviz.com` but that this will be more important to keep an eye on for `vitals`.

Reply:
Sounds like we've got the tooling in place to manage this.